### ISSUE TITLE
Raw is not kept as selection when going over a different item on the first list

### ISSUE THREAD

**Issue by Ducasse** [pharo-spec/NewTools#211]
- I have a list of blocks which contains instructions. 
- I click on the first block
- I click on the raw pain of the block pane
- Now when I click on the second item of the list blocks, the raw (previous selection) is not kept but the first tab is shown instead.

**Comment by estebanlm** [comment]
the problem with this is then I will have always the raw selection (and not the default)
see, if inspect a class I see this: 

![image](https://user-images.githubusercontent.com/513630/115879349-d818bd80-a449-11eb-9a96-886b4d9be016.png)
 
then I want to click in "methodDict", and what I should see immediately is: 

![image](https://user-images.githubusercontent.com/513630/115879484-fed6f400-a449-11eb-8fd7-42bedfe89f98.png)

now, if I select the same as

**Comment by estebanlm** [comment]
sorry, didn´t meant to close it :)

**Comment by Ducasse** [comment]
Yes for class I can understand. I was talking about browsing a list of the same elements. 
When I click the next elements of the list, it refers to the same class, I had to systematically reclick and click and click on raw.

**Comment by Ducasse** [comment]
So I do not get how the example on browsing a class is relevant. I do not see why when you browse the same kind of object than the previous one we cannot simply keep the previous selected pane.

**Comment by estebanlm** [comment]
I will show you in person, is hard to explain by a github mail thread :)
(anyway I am not saying this is not solvable, I was just explained the problem I encounter today while trying to fix it)

**Comment by Ducasse** [comment]
Ok. 
Now to me the solution is that the behavior (keeping the last selected pane) should be linked to 
- 1 the fact that the same kind of object is browsed, 
- 2 then if this is not the case that the same pane exists in the inspector of the different class
- 3 if none of the case exist then the default order.
